packages/clay-tabs/src/TabPane.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React, {useEffect, useState} from 'react';
+
+interface ITabPaneProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Flag to indicate if `active` classname should be applied
+	 */
+	active?: boolean;
+
+	/**
+	 * Flag to indicate if `fade` classname that applies an fading animation should be applied.
+	 */
+	fade?: boolean;
+
+	/**
+	 * TabPane's key.
+	 */
+	tabkey: number;
+}
+
+const delay = (fn: Function) => {
+	return setTimeout(() => {
+		fn();
+	}, 100);
```

Comment:
Where did this value come from? Could it be moved into a named constant?

Reply:
Sure, I was minding about turn this delay a property. Makes sense?

/cc @bryceosterhaus @matuzalemsteles too

Reply:
I prefer to keep it away from the API, because this is just a fake time to cause the animation, developers don't need to know that.